src/components/Header/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import "./Header.css";
```

Comment:
If you're not using one already, consider using a formatter like `Prettier` to give the code a cleaner and easier-to-read appearance.

Reply:
Fixed